postgres/package.json

```
@@ -31,7 +31,7 @@
     "test": "npm run setup && jest --silent"
   },
   "dependencies": {
-    "@cap-js/db-service": "^1.0.1",
+    "@cap-js/db-service": "^1.1.0",
```

could we also do smt like this, so that we dont have to bump the minor version each time?
```suggestion
    "@cap-js/db-service": "^1",
```